src/nrrd/nrrdEnums.h

```diff
@@ -53,6 +53,7 @@ enum {
   nrrdIoStateZlibLevel,
   nrrdIoStateZlibStrategy,
   nrrdIoStateBzip2BlockSize,
+  nrrdIoStateChunckElementNumber,
```

Comment:
Is this used?

Reply:
No I removed, we could add method in nrrdIoStateSet to set the number of chunk and start element using this function: nrrdIoStateSet(nio, nrrdIoStateSkipData, 1); 

However the function nrrdIoStateSet takes in input only int and it is not big enough for the number of elements for large chunks. TO make less modification as possible to teem, I am not modifications the function nrrdIoStateSet. Anyway the nrrdIoState variables are public so there is not really the necessity to use the nrrdIoStateSet function.